### PR TITLE
docs: update the IPython Magics docs

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-magics/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)
     - docs can be viewed in: https://bigframes-swe.teams.x20web.corp.google.com/ashleyxu/html_magics_05/index.html

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ Unsupported Python Versions
 Python == 2.7, Python == 3.5, Python == 3.6.
 
 The last version of this library compatible with Python 2.7 and 3.5 is
-`google-cloud-bigquery==1.28.0`.
+`bigquery-magics==0.1.0`.
```
Comment (tswast) [inline README.rst:63]: Not true. We don't have any version with Python 2.7 support. Please remove these lines.

File: README.rst [hunk 2]
```diff
@@ -81,4 +81,26 @@ Windows
...
     virtualenv <your-env>
     <your-env>\Scripts\activate
-    <your-env>\Scripts\pip.exe install bigquery-magics
\ No newline at end of file
+    <your-env>\Scripts\pip.exe install bigquery-magics
+
+Example Usage
+-------------
+
+To use these magics, you must first register them. Run the ``%load_ext bigquery-magic``
```
Comment (tswast) [inline README.rst:89]: I believe this needs to match the import name, not the package name.

```suggestion
To use these magics, you must first register them. Run the ``%load_ext bigquery_magics``
```

See: https://github.com/googleapis/python-bigquery-magics/blob/23d54cd6721e362974f0119534fad726dc9a9608/samples/snippets/query_params_scalars.py#L27

File: README.rst [hunk 3]
```diff
@@ -81,4 +81,26 @@ Windows
...
+in a Jupyter notebook cell.
+
+.. code-block::
+
+    %load_ext bigquery-magics
```
Comment (tswast) [inline README.rst:94]:
```suggestion
    %load_ext bigquery_magics
```

File: docs/index.rst [hunk 4]
```diff
@@ -18,21 +9,12 @@ More Examples
...
-  reference
-  dbapi
-  design/index
 
 Migration Guide
 ---------------
 
-See the guides below for instructions on migrating from older to newer *major* releases
-of this library (from ``1.x`` to ``2.x``, or from ``2.x`` to ``3.x``).
+Migrating from the ``google-cloud-bigquery``, you need to run the ``%load_ext bigquery-magics``
```
Comment (tswast) [inline docs/index.rst:16]: This reads strangely. Please show a "before" and "after" code sample.

File: docs/magics.rst [hunk 5]
```diff
@@ -6,7 +6,7 @@ in a Jupyter notebook cell.
 
 .. code::
 
-    %load_ext google.cloud.bigquery
+    %load_ext bigquery-magics
```
Comment (tswast) [inline docs/magics.rst:9]:
```suggestion
    %load_ext bigquery_magics
```

**Review by tswast — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (ashleyxuu) [reply README.rst]: Done.

File: docs/index.rst (hunk 4 above)
Comment (ashleyxuu) [reply docs/index.rst]: Done.

File: docs/magics.rst (hunk 5 above)
Comment (ashleyxuu) [reply docs/magics.rst]: Done.

**Review by tswast — APPROVED** [review]
Thanks!

**Pull request merged**